src/main/java/net/william278/velocitab/packet/ScoreboardManager.java

```diff
@@ -107,7 +164,7 @@ public void updateRole(@NotNull Player player, @NotNull String role) {
 
 
     public void resendAllNameTags(Player player) {
-        if (!plugin.getSettings().doNametags()) {
+        if (!plugin.getSettings().isSortPlayers()) {
```

Comment:
No I think we want to keep `doNametags()` for this one!

Reply:
The sorting system is handled with teams. Maybe the problem is with the method name, I would change that to resendAllTeams

Reply:
You should check the Settings#doScoreboardTeamsPackets setting instead